_drafts/2023-05-11-ota-for-embedded-linux-devices.md

```diff
@@ -0,0 +1,446 @@
+---
+title: "OTA for Embedded Linux Devices: A practical introduction"
+author: tsarlandie
+tags: [linux, ota, swupdate]
+---
+
+<!-- excerpt start -->
+
+A core belief of Memfault is that we can ship faster when we have efficient
+tools to detect, debug and fix issues in the field. Fixing issues on your
+computer is not enough: the fix must reach the end users!
+
```

Comment:
Intros are truly the most difficult thing. 

I tried my hand at one, but bailed after a few minutes: 

Trying out a new suggestion for an intro:

A core belief of ours at Memfault is that over-the-air updates (OTA) for devices is a requirement, and it's often one of the items we recommend development teams build out first. With an OTA system in place, companies can quickly ship their product to customers and rapidly adapt their product to customer demands. A great OTA system can be a life-saver, and a bad one can wreak havoc on a company....something something. ChatGPT to the rescue I guess.